.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+* @FabricMC/developers
```

Comment:
Not sure if this is intended.

Reply:
This was a copy from yarn, it might be ok once merged?

Reply:
Probably, could just be because its from a fork :+1: